server/validators/isValidExclusionZoneFile.ts

```diff
@@ -4,13 +4,12 @@ import { isBlank } from '../utils/utils'
 
 export default function IsValidExclusionZoneFile(validationOptions?: ValidationOptions) {
   const isValidExclusionZoneFile = (outOfBoundFilename: string, { object }: ValidationArguments) => {
-    const uploadFile = _.get(object, 'uploadFile')
+    const { uploadFile } = object as Record<string, Record<string, unknown>>
     // If there is a file upload present in the request then validate it
     if (uploadFile) {
       return (
         uploadFile.fieldname === 'outOfBoundFilename' &&
         uploadFile.originalname === outOfBoundFilename &&
-        uploadFile.size > 0 &&
```

Comment:
still need this check 

you should be able to case it as a number 

`(uploadFile.size as number) > 0`

or parse it:

`parseInt(uploadFile.size?.toString()) > 0`

Reply:
Done.